README.md

```diff
@@ -222,8 +222,35 @@ It returns the same type as the `list_like` argument
 ```
 
 
+#### `apply_formatters_to_sequence(formatters, <list_like>)` -> <new_list_like>
+
+This function will apply each formatter at to the list-like value, at the
+position it was supplied. It returns the same time as the `list_like` argument.
+For example:
+
+```py
+>>> from eth_utils.curried import apply_formatters_to_sequence
+
+>>> list_formatter = apply_formatters_to_sequence([bool, int, str])
+
+>>> list_formatter([1.2, 3.4, 5.6])
+[True, 3, '5.6']
+
+>>> list_formatter((1.2, 3.4, 5.6))
+(True, 3, '5.6')
+
+# it will pass through items longer than the number of formatters supplied
```

Comment:
This doesn't seem like good behavior.  I think it should fail if it runs out of formatters.